src/integrationTest/java/com/mongodb/kafka/connect/avro/TweetMsg.java

```diff
@@ -72,7 +72,7 @@ public static TweetMsg fromByteBuffer(
         return DECODER.decode(b);
     }
 
-    private long _id;
+    private long id;
```

Comment:
Minor change to ensure that inserting a TweetMsg will create an object id, this allows testing of insertion order.